### PR TITLE
export "domUtils.isElementReadOnly"

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @churkin, @miherlosev

### CODE REVIEW

File: src/client/utils/dom.js [hunk 1]
```diff
@@ -629,6 +629,10 @@ export function isFetchRequest (instance) {
     return instance && instanceToString(instance) === '[object Request]';
 }
 
+export function isElementEditingAllowed (el) {
```
Comment (miherlosev) [inline src/client/utils/dom.js]: `IsElementReadOnly`?

Comment (georgiy-abbasov) [inline src/client/utils/dom.js]: Really

File: test/client/fixtures/utils/dom-test.js [hunk 2]
```diff
@@ -694,17 +694,17 @@ test('isTextEditableInput', function () {
...
 
-    ok(domUtils.isElementEditingAllowed(input));
+    ok(!domUtils.isElementReadOnly(input));
 
     input.readOnly = true;
 
-    ok(!domUtils.isElementEditingAllowed(input));
+    ok(domUtils.isElementReadOnly(input));
 
     input.readOnly = false;
```
Comment (miherlosev) [inline test/client/fixtures/utils/dom-test.js]: Add `ok(!domUtils.isElementReadOnly(input));` check there

File: test/client/fixtures/utils/dom-test.js [hunk 3]
```diff
@@ -691,6 +691,24 @@ test('isTextEditableInput', function () {
     }
 });
 
+test('isElementEditingAllowed', function () {
```
Comment (miherlosev) [inline test/client/fixtures/utils/dom-test.js]: Rename test too

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Pull request merged**